userspace/libsinsp/plugin.cpp

```diff
@@ -591,6 +591,20 @@ bool sinsp_plugin::resolve_dylib_symbols(std::string& errstr) {
 		                           m_handle->api.get_async_events,
 		                           m_async_event_names,
 		                           false);
+
+		// Register each async event name in sinsp global event table.
+		for(const auto& name : m_async_event_names) {
```

Comment:
When loading an ASYNC cap plugin, register all of its `async_event_names` in the global `g_infotables.m_event_info` table.